scripts/scripts.js

```diff
@@ -42,6 +42,12 @@ export function getDateFromExcel(date) {
   return date;
 }
 
+export const formatDate = (date) => date.toLocaleDateString('en-US', {
```

Comment:
is this not needed for other locales ?

Reply:
if we wanted to fix their site, yes. However, the existing site formats in the en-US format for all langs/locales.